examples/canvas/build.sh

```diff
@@ -4,9 +4,9 @@ if [ ! -z ${USE_LINKER+x} ]; then
   echo "Linker is $USE_LINKER"
 fi
 
-ghc -Wall -i../.. \
+ghc -XCApiFFI -Wall -i../../src \
```

Comment:
Leave it as is, I'll walk through the bash scripts on my own to save you some time.

I've tested that running examples with `cabal` such as `cabal run counter` works well and it's very convenient to switch between examples this way. It's still useful to keep the scripts because using `GHC` directly provides faster iteration loop. As an example, time to rebuild after a single line change
```
$ time cabal build spaceShooter
real    0m4.217s
user    0m8.526s
sys    0m3.729s

$ time ./build.sh
real    0m1.460s
user    0m3.675s
sys    0m1.533s
```